platform/core/src/classes/metadata/StudyMetadata.js

```diff
@@ -281,13 +293,14 @@ export class StudyMetadata extends Metadata {
 
     // Loop through the series (SeriesMetadata)
     this.forEachSeries(
-      series =>
-        void this._createDisplaySetsForSeries(
-          sopClassHandlerModules,
-          series,
-          displaySets
-        )
-    );
+      series => {
+         const displaySetsForSeries = this._createDisplaySetsForSeries(
+           sopClassHandlerModules,
+           series,
+         );
+
+         displaySets.push(...displaySetsForSeries);
```

Comment:
I like this better than passing the array and pending in the fn. Less side-effecty 👍